main/src/main/java/cgeo/geocaching/unifiedmap/UnifiedMapViewModel.java

```diff
@@ -53,6 +56,11 @@ public void setCurrentPositionAndHeading(final Location location, final float he
         positionAndHeading.setValue(new Pair<>(location, heading));
     }
 
+    /** null if coords indicator should be hidden */
+    public void setCoordsIndicator(@Nullable final Geopoint geopoint) {
```

Comment:
This setter method might not be necessary here. You could just call `viewmodel.coordsIndicator.setValue(geopoint);` from inside UnifiedMapActivity... At least that's the way how I did it at most other places. Explicit setter methods inside the viewmodel should IMHO just be used if there is more processing involved but not for simple pass through.

Reply:
ah, that's even easier, thanks for pointing this out. I'm still in the process of getting used to the new ViewModel for UnifiedMap...

Reply:
> ah, that's even easier, thanks for pointing this out. I'm still in the process of getting used to the new ViewModel for UnifiedMap...

Really appreciated! Don't hesitate to ask if any questions should arise :)